src/content/en/updates/2017/02/navigation-preload.md

```diff
@@ -0,0 +1,338 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Navigation preload lets you overcome service worker startup time by making requests in parallel.
```

Comment:
Knowing that 'improve' doesn't work, I also know that I don't understand what you're conveying here. Are you circumventing start up time? Are you expediting it?